uber/templates/preregistration/preregbase.html

```diff
@@ -69,7 +71,7 @@
                 extra: 0
             }]
         };
```

Comment:
We always want to show the tshirt row for staffers and not make them pay extra.  I considered putting this into the MagClassic repo instead of in core, but I imagine that most events provide tshirts for their staffers, so this seems like a sensible default.